src/main/java/org/broadinstitute/hellbender/utils/read/GATKRead.java

```diff
@@ -73,6 +73,21 @@ default boolean isEmpty() {
     void setPosition( final Locatable locatable );
 
     /**
+     * @return The actual contig assigned to the read, regardless of unmapped status. Unlike {@link #getContig}, will
```

Comment:
too many negatives here "Unlike .. will return non-null". It is true to say that this method never returns null?